lib/tasks/locale.rake

```diff
@@ -91,7 +91,7 @@ namespace :locale do
 
     yamls.each_key do |yaml_glob|
       yaml_glob_full = root_path.join(yaml_glob)
-      Dir.glob(yaml_glob_full).each do |file|
+      Dir.glob(yaml_glob_full).sort.each do |file|
```

Comment:
These aren't really necessary since we sort by msgid later but it's annoying to see Dir.glob without sort.